app/src/main/res/values/strings.xml

```diff
@@ -5,7 +5,6 @@
   <string name="menu_home">Home</string>
   <string name="menu_options">Options</string>
   <string name="menu_my_downloads">My Downloads</string>
-  <string name="menu_help">Help</string>
   <string name="exploration_activity_title">Exploration Player</string>
   <string name="help_activity_title">Help</string>
```

Comment:
Could you check if help activity is using this or not?

Also, are there any other items which we need to put under help activity.

Reply:
Yes, I've checked it and I've also searched this and found no use in the help activity.
Also, I've searched all items, and only the above all I've found that were used in help activity